bark/models/behavior/behavior_rss/behavior_rss.hpp

```diff
@@ -161,9 +182,11 @@ class BehaviorRSSConformant : public BehaviorModel {
   std::shared_ptr<BaseEvaluator> rss_evaluator_;
   BehaviorRSSConformantStatus behavior_rss_status_;
   double world_time_of_last_rss_violation_;
+  State last_state_;
```

Comment:
why do we set a last_state_here? should that not be stored in the agent's history?

I think this also leads to errors, such as checking if the time is valid. In the observed world we should get the agent's state.